scenarios/multiple_ERC20.js

```diff
@@ -0,0 +1,74 @@
+import eth from 'k6/x/ethereum';
+import exec from 'k6/execution';
+import { fundTestAccounts } from './helpers/init.js';
+import { textSummary } from 'https://jslib.k6.io/k6-summary/0.0.2/index.js';
+
+export const options = {
+    setupTimeout: '220s',
+    scenarios: {
+        constant_request_rate: {
+            executor: 'constant-arrival-rate',
+            rate: 200,
+            timeUnit: '1s',
+            duration: '1m',
+            preAllocatedVUs: 10,
+            maxVUs: 10,
+        },
+    },
+};
+
+// You can use an existing premined account
+const root_address = "0x1AB8C3df809b85012a009c0264eb92dB04eD6EFa";
+const mnemonic = __ENV.LOADTEST_MNEMONIC;
+let rpc_url = __ENV.RPC_URL;
+if (rpc_url == undefined) {
+    rpc_url = "http://localhost:10002"
+}
+
+const ZexCoin = JSON.parse(open("./contracts/ZexCoinERC20.json"));
+
+export function setup() {
+    let data = {};
+
+    const client = new eth.Client({
+        url: rpc_url,
+        mnemonic: mnemonic,
+    });
+
+    const receipt = client.deployContract(JSON.stringify(ZexCoin.abi), ZexCoin.bytecode.substring(2), 500000000000, "ZexCoin", "ZEX")
+
+    return {
+        accounts: fundTestAccounts(client, root_address),
+        contract_address: data.contract_address
+    };
+}
+
+let nonce = 0;
+let client;
+
+// VU client
```

Comment:
Just curious what does VU abbreviation stand for?

Reply:
Virtual User